_posts/2013-07-17-using-caption-services-with-html5-video.md

```diff
@@ -1,19 +1,28 @@
 ---
 layout: post
-title: "How-to: Using caption services with HTML5 video"
+title: "How-to: Using captions with HTML5 video"
 description: "How to implement captions on HTML5 video (and audio) elements."
 published: true
 author: greg_tarnoff
 categories:
   - How-tos
+further_reading:
+  - url: https://www.3playmedia.com/solutions/how-it-works/how-to-guides/html5-video-captioning/
+    title: "How to Add Closed Captions & Subtitles to HTML5 Videos"
+    source: 3PlayMedia
+  - url: https://support.google.com/youtube/topic/9257536
+    title: "YouTube Captions"
+    source: YouTube Help
+  - url: https://www.speechpad.com/captions/ttml
+    title: TTML (Timed Text Markup Language)
+    source: Speechpad
 ---
 
 One of the great things about the internet is the ability to share video. With HTML5 we can now use video without the use of Flash. This makes video more accessible as the controls are located in the DOM. However, to do accessible video right, you need to include captions and transcripts.
 
-In addition to making video accessible to those with hearing issues, having a transcript on the page can provide you localization of a video not always available in the local language. With a transcript on your page, you may increase your search engine rankings by making your content easily indexed by search engine crawl spiders.
-
-So how do we pull off putting captions into a video after the video has been produced? Fortunately, the HTML5 video tag has a solution for us. We have to provide a transcript file in either XML or a VTT as a track element after your video source files.
+In addition to making video accessible to those with hearing issues, having a transcript on the page can provide you localization of a video not always available in the local language. With a transcript on your page, you may increase your search engine rankings by making your content easily indexed by search engine crawl spiders. Transcripts also give all users an alternate way to access the information presented in a video.
 
+So how do we pull off putting captions into a video after the video has been produced? Fortunately, the HTML5 video tag has a solution for us. We have to provide a transcript file as a track element after the video source files.
```

Comment:
As a bit of additional cleanup, let's replace the text "HTML5 video tag" with "video element".  

The copy correctly reference the "track element" later in the same paragraph.   Additionally, HTML5 isn't a spring chicken anymore.  We can just say "HTML" at this point.